src/backend/_scss/application.scss

```diff
@@ -411,8 +411,18 @@ a {
 /* Track SCSS */
 
 .show-button-filter {
-  display: inline-block;
-  padding-top: 6px;
+  padding-top: 10px;
```

Comment:
Are you sure going to Flex-box approach. I doubt what will happen when Date-filters are increased.
Flex will try to make it in single line which makes the font on mobile invisible.